pandas/tests/arrays/categorical/test_repr.py

```diff
@@ -37,7 +37,6 @@ def test_big_print(self):
     def test_empty_print(self):
         factor = Categorical([], ["a", "b", "c"])
         expected = ("[], Categories (3, object): [a, b, c]")
-        # hack because array_repr changed in numpy > 1.6.x
         actual = repr(factor)
         assert actual == expected
```

Comment:
Fair enough to say that the repr of np.array has arrived, no? no need to call getting a list from it a hack, IMO